plugin/cog/cog-rangereader-s3/src/main/java/it/geosolutions/imageioimpl/plugins/cog/S3ConfigurationProperties.java

```diff
@@ -40,11 +40,11 @@
     private final static String S3_DASH = "s3-";
 
     /** Virtual-Hosted-Styles URL pieces */
-    private final static String S3_DOT_VH = "." + S3_DOT;
+    public final static String S3_DOT_VH = "." + S3_DOT;
```

Comment:
Not sure why these needed to become public... seems like package-private would have done the trick anyways? Maybe you need them downstreams too?